var.tfvars

```diff
@@ -47,6 +47,7 @@ cluster_id_prefix = "test-ocp" # Set it to empty if just want to use cluster_id
 cluster_id        = ""         # It will use random generated id with cluster_id_prefix if this is not set
 
 use_zone_info_for_names = true # If set it to false, the zone info would not be used in resource names on PowerVS.
+fips_compliant      = false
```

Comment:
This should be commented out, as by default it is already false. Also add a comment as seen for other options.

Reply:
Done